.ci/travis.sh

```diff
@@ -90,7 +90,7 @@ checkstyle-tester-diff-groovy-regression-single)
   export MAVEN_OPTS="-Xmx2048m"
   groovy ./diff.groovy --listOfProjects projects-to-test-on.properties \
     -pc ../../../checkstyle-tester/diff-groovy-regression-config.xml \
-    -r ../../checkstyle -xm "-Dcheckstyle.failsOnError=false" \
+    -r ../../checkstyle \
```

Comment:
CLI doesn't have anything similar. It seems we need to build something into groovy.

> If this is true, and Checkstyle reported any violations or errors, the build fails immediately after running Checkstyle

The only way we have to determine if any violations/errors are reported from the CLI is the exit code. Anything else requires adding custom modules to listen for violations.

Reply:
I added the reverse option, `continueOnError`. It is inverted because we need the default to be `false` so when we use the option, it is enabled. It follows the POM's default value which was `failsOnError = true` so `continueOnError = false` by default.